### PR TITLE
Fixing issue with page images in Safari

### LINKED ISSUE #112: Page images browser compatibility
The hero image displays differently between browsers in larger (desktop) screens.

The image appears correctly in Chrome and Firefox but in Safari, it does not. See screenshot:
Screen Shot 2020-10-04 at 12 06 40 PM

The next image below is also having the same issue

Screen Shot 2020-10-04 at 12 12 28 PM

### PR DESCRIPTION
**Pull request by viki-korotkaya**
Closes #112

## What is the story, issue or feature this Pull Request is solving or implementing?
Fixed issue with page images displaying page images in Safari.

## Completeness Checklist
- [ ] Local Tests are passing.
- [ ] New tests have been added for new code where relevant
- [ ] Screenshots of before and after changes have been added
- [ ] JSDoc Documentation and comments have been added or updated in the code.
- [ ] Documentation in the /docs folder has been added or updated when appropirate.
- [ ] Follows Nouri JavaScript guidelines
- [ ] Commit message follows Nouri guidelines
- [ ] Introduces minimal or no additional external packages.

Closes #<112>

### CODE REVIEW

File: src/components/home/sections/About.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const About = () => {
     <section id='about'>
       <h2>{HEADER_TEXT}</h2>
       <div className='about-content-container'>
-        <img id='about-groceries' src={groceries} />
+        <div className='about-groceries-container'><img id='about-groceries' src={groceries} /></div>
```
Comment (ambergkim) [inline src/components/home/sections/About.js:16]: Interesting way of wrapping the image in a container. :)

**Comment by ambergkim** [comment]
The first comment above

Not sure if this will work
```
Closes #<112>
```

This will definitely work. Once the PR is merged, it will automatically close the issue related to it.
```
Closes #112
```

**Comment by ambergkim** [comment]
For future PR's also, it would be good to put before and after screenshots of your work and detail why there is the issue, your approach, etc. 🙂

Otherwise, this looks good.

**Review by ambergkim — APPROVED** [review]

**Comment by ambergkim** [comment]
Also, future commits:

```
Fixing issue with page images in Safari

Changes:
- details

Closes #112
```

Rather than
```
Fixing issue with page images in Safari
Closes 112
```

I think the formatting needs the #122 to be seen and there might be issues sometimes if there is no empty line after the title.

File: src/components/home/sections/About.js (hunk 1 above)
Comment (viki-korotkaya) [reply src/components/home/sections/About.js:16]: :) Sorry, I will separate them

**Pull request merged**